lib/sql.js

```diff
@@ -1508,3 +1508,251 @@ SQLConnector.prototype.getInsertedId = function(model, info) {
 SQLConnector.prototype.executeSQL = function(sql, params, options, callback) {
   throw new Error(g.f('{{executeSQL()}} must be implemented by the connector'));
 };
+
+// Refactored Discovery methods
+
+SQLConnector.prototype.buildQuerySchemas = function(options) {
+  return 'INFORMATION_SCHEMA';
```

Comment:
This should be a SQL statement that queries `INFORMATION_SCHEMA` for schema names.